CONTRIBUTING.md

```diff
@@ -31,7 +31,7 @@ why we ask this as well as instructions for how to proceed, see the
   cd citus
   ./configure
   make
-  make install
+  make install-all
```

Comment:
why  `make install`  doesn't install the downgrade packages?

Reply:
We don't want packagers to include the downgrade scripts in their packages, since it will cause conflicts with the `citus-downgrade` package we want to provide later.

Reply:
Wouldn't we end up with the same situation if we ask users to do `make install-all`?

Reply:
yes, see my other comment. The difference should be explained here.